extensions/azure/processors/FetchAzureBlobStorage.h

```diff
@@ -0,0 +1,72 @@
+/**
+ * @file FetchAzureBlobStorage.h
+ * FetchAzureBlobStorage class declaration
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <memory>
+#include <optional>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "core/Property.h"
+#include "AzureBlobStorageProcessorBase.h"
+#include "core/logging/LoggerConfiguration.h"
+
+template<typename T>
+class AzureBlobStorageTestsFixture;
+
+namespace org::apache::nifi::minifi::azure::processors {
+
+class FetchAzureBlobStorage final : public AzureBlobStorageProcessorBase {
+ public:
+  // Supported Properties
+  EXTENSIONAPI static const core::Property RangeStart;
+  EXTENSIONAPI static const core::Property RangeLength;
+
+  // Supported Relationships
```

Comment:
The comment about properties is misleading, because there are more supported properties. The one about relationships is just redundant, because it's obvious from the context. Consider removing them.

Reply:
Removed in 1180ddd237dae38a7f000c250be3ec3cc3c025d4